osbs/utils/__init__.py

```diff
@@ -588,6 +590,17 @@ def retry(*args, **kwargs):
     return do_retry_on_exception
 
 
+def user_warning_log_handler(self, msg, *args, **kwargs):
+    def convert_warn_message(msg):
+        if isinstance(msg, dict):
+            return json.dumps(msg)
+
+        return msg
+
+    msg = convert_warn_message(msg)
```

Comment:
How is inner function `convert_warn_message` useful for this case?

Reply:
It's deprecated now